connectivity/http/http.c

```diff
@@ -327,16 +331,53 @@ static int request_log(void *cls, const struct sockaddr *addr, socklen_t addrlen
   return MHD_YES;
 }
 
+/*
+ * Append data from the end of ta_http_request_t.request
+ */
+static status_t build_request(ta_http_request_t *req, const char *data, size_t size) {
+  if (req == NULL || data == NULL) {
+    ta_log_error("Illegal NULL pointer\n");
+    return SC_TA_NULL;
+  }
+  if (size == 0) {
+    ta_log_error("Illegal data size : 0\n");
+    return SC_TA_NULL;
+  }
+  if (size > 0) {
```

Comment:
This `if` can be removed, the invalid params are handled.

Reply:
Yes.